tools/db_ping_remote.rb

```diff
@@ -1,10 +1,10 @@
+require File.expand_path('../config/environment', __dir__)
```

Comment:
Pedantic (possibly for another PR):  Would adding a `#!/usr/bin/env ruby` at the top here makes sense and making these executable?

I assume there is a reason they are not already, it would just cut down on the typing further:

`ruby tools/purge_orphaned_tag_values.rb -s 100` v.s. `tools/purge_orphaned_tag_values.rb -s 100`